package.json

```diff
@@ -111,13 +111,12 @@
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-react": "^7.11.1",
     "file-loader": "^1.1.11",
-    "html-webpack-plugin": "^3.2.0",
+    "html-webpack-plugin": "^4.0.0",
     "husky": "^1.1.2",
     "jest": "^24.8.0",
     "lint-staged": "^7.3.0",
     "mkdirp": "^0.5.1",
     "node-sass": "^4.9.0",
-    "path": "^0.12.7",
```

Comment:
There is a whole story about polyfills and old libraries hidden in this delete. We're running on node 10 + so we don't need to worry about it. The package itself is 5 years old. The reason we hadn't had a problem before is because webpack <5 came with polyfills for core node modules, but webpack 5 has removed that. We're polyfilling ourselves later on.